actors/builtin/power/policy.go

```diff
@@ -4,8 +4,10 @@ import (
 	abi "github.com/filecoin-project/specs-actors/actors/abi"
 )
 
-// Minimum number of registered miners for the minimum miner size limit to effectively limit consensus power.
-const ConsensusMinerMinMiners = 3
+// PARAM_SPEC
+// This is a switch for turning on the minimum miner size requirement.
+// When this number of miners of minimum miner size is reached, minimum miner size is enforce.
+const ConsensusMinerMinMiners = 4
```

Comment:
**More analysis needed**:
- This number cannot be 3, since 3 guarantees that there is a miner with at least >33% of the power
- This number should either be 0 (meaning that only miners with minimum miner size are allowed to mine)
- Or this number should be set to >4 (e.g. 10 or 100), to ensure that the network is "enough" decentralized
  - Note that this would give the ability for small miners power to participate in the consensus and block reward earning at the beginning. You can consider this as a "threshold" of network decentralization, after which minimum miner size is enforced

Note: This check does not guarantee anything about security, since below `ConsensusMinerMinMiners` we could have a single miner with ~99% and above this threshold we can still have a single miner with ~99%.

Todo: We need to make sure that when the minimum miner size switch is turned on, we don't cutoff many small miners that contribute to 67% security.

cc @irenegia, @jzimmerman, @lucaniz, @sa8